avm/res/recovery-services/vault/tests/e2e/max/main.test.bicep

```diff
@@ -341,15 +341,18 @@ module testDeployment '../../../main.bicep' = [for iteration in [ 'init', 'idem'
     }
     privateEndpoints: [
       {
+        ipConfigurations: {
+          name: 'myIPconfig'
+          properties: {
+            groupId: 'account'
+            memberName: 'default'
+            privateIPAddress: '10.0.0.10'
+          }
+        }
         privateDnsZoneResourceIds: [
           nestedDependencies.outputs.privateDNSZoneResourceId
         ]
         subnetResourceId: nestedDependencies.outputs.subnetResourceId
-        tags: {
```

Comment:
Not a fan of tags? :D Usually we have them in all `max` tests to show how to use the parameter.